### DISCUSSION TITLE
Newbie / Idling / Transcoding / Playlists

### DISCUSSION THREAD

**Discussion by JOJ0** [navidrome/navidrome#2326] [Q&A] [answered]
New user here. Very pleased with Navidrome so far. Love the focus on simplicity and performance! Tried airsonic/airsonic-advanced in docker on dated Synology NAS. Disastrous performance! With Navidrome I think I finally landed at the project suited for my needs and I'd like to support. Thanks!

I could figure out most of my requirements through the excellent documentation, some questions remain though.

1
As said, running on dated (~2010) NAS in the official deluan/navidrome-latest container. Synology DSM has a neet "autoshutdown on idle" feature that works quite reliable with there own tools (not all). Not sure about how it exactly detects idling but I suppose/hope it's a very low level thing sensing filesystem or even block device access.

So if I would configure Navidrome to not autoscan the lib periodically, put logging to a minimum (info?, warn?) what services would remain on N.'s end OR on the minimal docker image (alpine?) that could prevent an idling for let's say, an hour.

What could interfere because of periodically doing something?

2
How does Navidrome/Subsonic handle transcoding. What I'd like to achieve is only FLAC and WMA is transcoded. MP3 stays as-is. I'm only streaming/downloading inhouse and suppose/hope my phone can handle any mp3 encoded files already. I think I understood from fiddling with mobile subsonic apps that they can request what and when to transcode. Some clients have settings like "let the server decide". What would would I need to set on Navidrome's transcoding settings to provide the basis of my requirement (basically that is: dont try to transcode mp3, but do with the rest)?

3
Regarding playlists: I generate a lot with beets smartplaylist plugin. All imports fine to Navidrome. Basically those generated lists should stay readonly. Is there any way to tell a subsonic server to mark playlists as RO? I still can try to rearrange things or delete tracks from those playlists from clients. Provided the music dir is mounted to the container RW, would/should it be possible to rearrange things in a list and would it be saved permanently to the m3u file on-disk?

Second playlist question: A generated playlist from within N.: Is it possible to save it to the filesystem or is it only staying within Navidrom/its database? Had a look into the sqlite db and it seems they don't have a value for path field where those imported from the filesystem do have that field populated.

Probably not all of this is supposed to go and be questioned in a Navidrome only gh-discussion. Some might be broader or generally about Subsonic. Mixig things here, I know... Just trying to reach out and see what can be answered here before I move on with my research!

Thanks again for a great tool! :-)

**Comment by arsaboo** [comment]
You can set the transcoding for each player from the web. Settings > Players has all the players listed. Choose the player you want to configure and change the transcoding settings. If you set it to mp3, my understanding is everything will be transcoded to mp3 for that player. 
<img width="233" alt="image" src="https://user-images.githubusercontent.com/18319734/234267703-a65d4cdf-d545-471a-add3-c9ff2e5851c5.png">

**Comment by JOJ0** [comment, marked as answer]
I might have found the reason for issue 1) https://github.com/navidrome/navidrome/discussions/2271

Will try to disable the healthcheck by "composing" the container myself.
